yorkie/rpc/server.go

```diff
@@ -125,3 +136,27 @@ func (s *Server) listenAndServeGRPC() error {
 
 	return nil
 }
+
+// Validate validates port number and checks if files exists
+func (c *Config) Validate() error {
+	if c.Port < 1 || 65535 < c.Port {
+		return fmt.Errorf("must be between 1 and 65535, given %d: %w", c.Port, ErrInvalidRPCPort)
+	}
+
+	// when specific cert or key file are configured
+	if c.CertFile != "" {
+		_, err := os.Stat(c.CertFile)
+		if os.IsNotExist(err); err != nil {
+			return fmt.Errorf("%w: %s", ErrInvalidCertFile, c.CertFile)
+		}
```

Comment:
```suggestion
		if _, err := os.Stat(c.CertFile); err != nil {
			return fmt.Errorf("%s: %w", c.CertFile, ErrInvalidCertFile)
		}
```

Reply:
I thought you wanted to use `os.IsNotExist()` not `os.Stat()`. I'm sorry for misunderstanding. As you wrote, I think this is a better approach because `os.Stat()` can detect more errors and in that case `ErrInvalidCertFile` would be more proper than `ErrNoCertFile` or `ErrNoKeyFile`. Thank you.